app/views/newflow/login_signup/profile_newflow.html.erb

```diff
@@ -0,0 +1,175 @@
+<% @parent_col_id = "profile" %>
+
+<% content_for :head do %>
+  <%= javascript_include_tag  "profile" %>
+<% end %>
+
+<div id="temporary-logo-within-my-account">
+  <%= render(partial: 'layouts/main_menu') %>
+</div>
+
+<%# This styling here is temporary, WIP. %>
+<style>
+  body {
+    background: none;
+  }
+
+  #application-header {
+    display: none;
+  }
+
+  a, .editable-click, #enable-other-sign-in, a.sign-out, #add-an-email {
+    color: #027eb5 !important;
+  }
+
+  .ox-card {
+    background-color: #fff;
+    border: 0.1rem solid #d5d5d5;
+  }
+
+  #profile .ox-card .category {
+    color: var(--greyish-brown);
+  }
+
+  #application-footer {
+    display: none;
+  }
+
+  #newflow-footer {
+      font-size: 10px;
+      text-align: center;
+      margin-top: 8rem;
+  }
+
+  #newflow-footer--copyright a {
+    font-size: 10px;
+    color: #000 !important;
+  }
+
+  #footer-rice-logo-image {
+    margin-bottom: 1.2rem;
+  }
+
+  .ox-card {
+    height: 90rem;
+    width: 111rem;
+  }
+
+  .btn-primary {
+    background-color: #f36b32;
+    border: 0;
+  }
+
+</style>
```

Comment:
seems like this should be extracted to a `scss` file

Reply:
Oh, DEFINITELY. I am sorry for how ugly that code is, tbh 😅 . I plan to refactor it after release though for sure!